### ISSUE TITLE
Selecting a job when using the "join game" button does nothing.

### ISSUE THREAD

**Issue by Stormc12** [SS13-SCP13/SS13-SCP13#449]
#### Description of issue
When trying to join the game, clicking on the job button (scientist and scientist associate) does nothing, despite having the right rank in occupation settings.

#### Difference between expected and actual behavior
I expected to join the game as a scientist, but nothing happened.

#### Steps to reproduce 
1. Join server
2. Wait for round to start if it hasn't
3. Press "join game"
4. Attempt to select job.

#### Specific information for locating

It's in the join game menu

#### Length of time in which bug has been known to occur

First noticed it back when I was playing either last Friday or Saturday.

#### Client version, Server revision & Game ID

 2018-08-26
Master commit: 1bdfa7de98aaecc57d5c5ad42d066104ada3241a 
Local commit: 8e3288a692d078fd2d0a01f9e7d6da4e34ea985a

#### Issue bingo

Please check whatever applies. More checkboxes checked increase your chances of the issue being looked at sooner.

- [x] Issue could be reproduced at least once
- [ ] Issue could be reproduced by different players
- [x] Issue could be reproduced in multiple rounds
- [x] Issue happened in a recent (less than 7 days ago) round
- [x] [Couldn't find an existing issue about this]( https://github.com/SS13-SCP13/SS13-SCP13/issues )

**Comment by JakeDaBoss** [comment]
Was the job crossed out, or did it present any error messages?

**Comment by Stormc12** [comment]
No, the job wasn't crossed out, and it didn't bring up any error messages when I clicked on it.
